code/player.dm

```diff
@@ -20,10 +20,17 @@
 	var/list/cooldowns = null
 	/// position of client in in global.clients
 	var/clients_pos = null
+	/// the server time that this player joined the game, in 1/10ths of a second
+	var/round_join_time = null
+	/// the server time that this player left the game, in 1/10ths of a second
+	var/round_leave_time = null
+	/// the total time that this player has been playing the game this round, in 1/10ths of a second
+	var/current_playtime = null
```

Comment:
It's possible that this default value will be operated on in gameticker, and sent to the api. Please default to 0.

Reply:
i think i do null checks as opposed to !(thing). i can replace the value with 0 in the gameticker if its null, though.